ynput/OpenPype
# Hiero: thumbnail from multilayer exr

## Brief description
Hiero exctract thumbnail works on mutlilayer exr

## Description
Thumbnail creator needed to have specified `color` layer.

## Testing notes:
1. Open hiero workfile
2. add multilayer exr at timeline
3. Create publishable clip on the new clip
4. publish
5. see that all works well
